php73-fpm-dev/Dockerfile

```diff
@@ -11,4 +11,24 @@ LABEL org.label-schema.build-date=$BUILD_DATE \
   org.label-schema.vcs-url="https://github.com/skilld-labs/docker-php" \
   maintainer="Andy Postnikov <andypost@ya.ru>"
 
-RUN apk add --no-cache php7-pecl-xhprof
+RUN apk add --update php7-pecl-xhprof \
+  && apk del php7-pecl-xdebug \
+# build extra extensions
+  && apk add --virtual .php-build \
+#  --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing \
+  php7-dev php7-pear make gcc g++ \
+#  && sed -ie 's/-n//g' /usr/bin/pecl \
+  && CFLAGS="-Os -fomit-frame-pointer -g -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -fvisibility=hidden -Wall -Wno-strict-aliasing" \
+  CPPFLAGS="$CFLAGS" LDFLAGS="-Wl,-O1 -Wl,--hash-style=both -pie" \
+  pecl install xdebug-3.0.0beta1 \
+  && echo ';zend_extension=xdebug.so' > /etc/php7/conf.d/50_xdebug.ini \
```

Comment:
it's still have overhead so I disabled by default... but in SDC easy override CMD